README.md

```diff
@@ -74,7 +72,7 @@ a 2 different scenarios that influence our course of action.
 
 Below we have diagrams of specific cases that we handle.
 
-## Dependent Duilds Diagram
+## Dependent Builds Diagram
```

Comment:
You didn't like the portmanteau of "Dependent & Builds"? 😂

Reply:
I assumed it was a typo. 😆

Reply:
🤣